Ubuntu-Solutions-Engineering/openstack-installer
# Rework lxc networking - utopic

Several changes with single-installer's container creation to resolve issues found on a 14.10 host

At time of creating this PR, this hasn't been tested on trusty.

Note that when testing on utopic, this kernel is required to work around a bug installing uvtool:
https://bugs.launchpad.net/apparmor/+bug/1408833/comments/10

TO TEST:
1. start a single install.
2. verify that a container gets created, services get deployed, and kvms created and started.
